securedrop/sdconfig.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.ADJECTIVES = _config.ADJECTIVES
+        self.DATABASE_ENGINE = _config.DATABASE_ENGINE
+        self.DATABASE_FILE = _config.DATABASE_FILE
```

Comment:
This attribute is not always present and should be under the `else` of `if _config.ENGINE != 'sqlite'`.

Reply:
Ah, means it will be there as  empty string as default value.

Reply:
Actually i was wrong. This field will be present and have a valid value because `config.py` loads it correctly as `path.join(SECUREDROP_DATA_ROOT, 'db.sqlite')`. But we should still only load it if the engine is `sqlite`.